src/runtime_src/xdp/profile/device/device_intf.h

```diff
@@ -196,16 +202,17 @@ class DeviceIntf {
     DeadlockDetector*     mDeadlockDetector  = nullptr;
 
     /*
-     * Set bandwidth number to a reasonable default
+     * Set max bandwidths to reasonable defaults
      * For PCIE Device:
-     *   bw_per_lane = 985 MB/s (Wikipedia on PCIE 3.0)
-     *   num_lanes = 16/8/4 depending on host system
-     *   total bw = bw_per_lane * num_lanes
+     *   configuration: gen 3x16, gen 4x8 
+     *   encoding: 128b/130b
      * For Edge Device:
-     *  total bw = DDR4 memory bandwidth
+     *  total BW: DDR4 memory bandwidth
      */
-    double mMaxReadBW  = 9600.0;
-    double mMaxWriteBW = 9600.0;
+    double mHostMaxReadBW    = 15753.85;
```

Comment:
Weren't these values define in perf.cpp's function: xclGetHostWriteMaxBandwidthMBps().

Instead of defining the same "magic" value in to separate locations, which can lead to furture issue (e.g., one changes and the other isn't).  Could this code be refactor to define it in only one location.  

Some ideas:
1. Make these magic values a constexpr and use it as a return value for the perf.cpp function and initialization value for these variables.
2. User the perf.cpp functions to initialize these variables.

Reply:
cleaned up in next release